rio_tiler/utils.py

```diff
@@ -298,6 +303,18 @@ def _tile_read(
     )
 
     vrt_transform, vrt_width, vrt_height = get_vrt_transform(src_dst, bounds)
+    out_window = None
```

Comment:
also here can we set `out_window` to `out_window = windows.Window(col_off=0, row_off=0, width=vrt_width, height=vrt_height)`